packages/suite/types/dsl.ts

```diff
@@ -38,10 +38,10 @@ test('a test')
   })
 
 // Context Async
-test('a test')
+let t1 = test('a test')
   // should not be able to return non-object
   // $ExpectError
-  .step('return nonsense', async () => {
+  t.step('return nonsense', async () => {
```

Comment:
I had to break these expressions apart since with the overloaded `step` method, typescript was putting the location of the error on the `test()` invocation and not the `.step()` invocation and so in order to ensure that the correct error was being tested, I needed to force the error to only possibly be where expected